src/main/kotlin/nebula/plugin/compile/JavaCrossCompilePlugin.kt

```diff
@@ -45,7 +46,11 @@ class JavaCrossCompilePlugin @Inject constructor(private val providerFactory: Pr
                 val location by lazy { targetCompatibility.locate(project, providers) }
                 withType(JavaCompile::class.java) {
                     if (JavaVersion.current() >= JavaVersion.VERSION_1_9) {
-                        it.options.compilerArgs.addAll(listOf("--release", targetCompatibility.majorVersion))
+                        if (project.gradle.versionLessThan("6.6-rc-1")) {
+                            it.options.compilerArgs.addAll(listOf("--release", targetCompatibility.majorVersion))
+                        } else {
+                            it.options.release.set(targetCompatibility.majorVersion.toInt())
```

Comment:
use `CompileOptions.release` property with Gradle 6.6-rc-1 +

Reply:
Nice!

Reply:
The updated docs, for our future information: [6.6-rc-2 CompileOptions](https://docs.gradle.org/6.6-rc-2/dsl/org.gradle.api.tasks.compile.CompileOptions.html)